.github/workflows/cd.yml

```diff
@@ -28,6 +28,7 @@ jobs:
   deploy:
     needs: build
     runs-on: ubuntu-latest
+    environment: production
```

Comment:
added main as the deploy branch and added us as required reviewers

![SCR-20220311-k3x](https://user-images.githubusercontent.com/360826/157957868-4abf96dd-6f65-4e07-bfbd-9f83fc04c000.png)

Reply:
it's not case sensitive, it felt weird to use `Production` here
https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#creating-an-environment